src/main/java/core/basesyntax/DrawAble.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface DrawAble {
```

Comment:
Consider renaming the interface to `Drawable` for better readability and consistency with naming conventions 📚